packages/server-wallet/src/wallet/__test__/integration/create-ledger-channel.test.ts

```diff
@@ -0,0 +1,80 @@
+import {constants} from 'ethers';
+
+import {Channel} from '../../../models/channel';
+import {Wallet} from '../..';
+import {createChannelArgs} from '../fixtures/create-channel';
+import {seedAlicesSigningWallet} from '../../../db/seeds/1_signing_wallet_seeds';
+import {defaultTestConfig} from '../../../config';
+import {alice, bob} from '../fixtures/participants';
+import {DBAdmin} from '../../../db-admin/db-admin';
+
+let w: Wallet;
+beforeEach(async () => {
+  w = new Wallet(defaultTestConfig);
+  await new DBAdmin(w.knex).truncateDB();
+});
+
+afterEach(async () => {
+  await w.destroy();
+});
+
+describe('happy path', () => {
+  beforeEach(async () => seedAlicesSigningWallet(w.knex));
+
+  it('creates a ledger channel', async () => {
+    expect(await Channel.query(w.knex).resultSize()).toEqual(0);
+
+    const {participants, allocations} = createChannelArgs();
+
+    const createPromise = w.createLedgerChannel({participants, allocations});
+
+    await expect(createPromise).resolves.toMatchObject({
+      outbox: [
+        {
+          params: {
+            recipient: 'bob',
+            sender: 'alice',
+            data: {
+              signedStates: [{turnNum: 0}],
+              objectives: [
+                {
+                  participants: [alice(), bob()],
```

Comment:
Are these wallet core participants or api participants?

Reply:
`wallet-core` though in this context it doesn't matter; the only different is that `{ _isDestination: void }` is added on the typescript type for `wallet-core`

Reply:
I thought there was an actual runtime difference: I think we get zero-padding to bytes32 in wallet core but not in the api type? There's a chance that could fall foul of our runtime type validation?

Reply:
Hm, well, inside the wallet we only every use the `wallet-core` variant and it's what is placed in the DB. The only place the API variant is used is in the wallet API handlers which immediately convert them into the wallet-core variant, so I don't think this an issue right now and at the least it's not an issue in this PR

Reply:
My point is that we are fixing types on the request side of the API (e.g. #2808 ) and we should also fix them on the response side. AFAICT we are spitting out wallet-core types in the return value of an API call (assuming this test passes and is sensitive to that).